frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -5,19 +5,29 @@
   import t from '$lib/i18n';
   import { TitlePage } from '$lib/layout';
   import { z } from 'zod';
-  import { _createProject } from './+page';
+  import { _createProject, _projectCodeAvailable } from './+page';
   import AdminContent from '$lib/layout/AdminContent.svelte';
   import { useNotifications } from '$lib/notify';
-  import { Duration } from '$lib/util/time';
+  import { Duration, debounce } from '$lib/util/time';
   import { getSearchParamValues } from '$lib/util/query-params';
   import { isAdmin } from '$lib/user';
   import { onMount } from 'svelte';
+  import { browser } from '$app/environment';
 
   export let data;
   $: user = data.user;
 
   const { notifySuccess } = useNotifications();
 
+  let codeValidation: z.ZodType = z.string().toLowerCase().min(4, $t('project.create.code_too_short'));
+  if (browser) {
+    codeValidation = z.string().refine((code) => {
```

Comment:
it looks like this is changing the validation and it no longer has a min length. shouldn't this be `codeValidation = codeValidation.refine`?